tests/php/Security/GroupTest.php

```diff
@@ -136,6 +137,38 @@ public function testCollateAncestorIDs()
         $this->assertContains($orphanGroup->ID, $orphanGroup->collateAncestorIDs());
     }
 
+    /**
+     * Test that Groups including their children (recursively) are collated and returned
+     */
+    public function testCollateFamilyIds()
+    {
+        $group = $this->objFromFixture(Group::class, 'parentgroup');
+        $this->assertSame([2, 3, 4], $group->collateFamilyIDs());
```

Comment:
You mustn't hard code IDs because postgres doesn't seem to reset the IDs for each test.

You could use `$this->allFixtureIDs('Group')` perhaps?